packages/swapper/src/swappers/LifiSwapper/endpoints.ts

```diff
@@ -48,7 +48,7 @@ export const lifiApi: SwapperApi = {
 
     return tradeQuoteResult.map(quote =>
       quote.map(tradeQuote => {
-        const { selectedLifiRoute } = tradeQuote
+        const selectedLifiRoute = tradeQuote.steps[0]?.lifiSpecific?.lifiRoute
```

Comment:
Any reason why this lives at steps[0] and not top-level ?

Reply:
See https://github.com/shapeshift/web/pull/9653#discussion_r2128806951